cellxgene_schema_cli/cellxgene_schema/ontology.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, species: SupportedOrganisms):
             duplicated_gene_labels = set()
 
             for gene in genes:
-                gene = gene.rstrip().split(",")
```

Comment:
i'm not entirely sure why this file was picked up from running mypy against the `scripts/` directory. my best guess is that there's something within the `scripts/` directory that's importing this in, though i couldn't find a direct import

Reply:
Your theory re:import dependency sound right. Not a big deal either way. If you feed explicit file names to mypy instead of packages or modules, it may not do the transitive checking (https://mypy.readthedocs.io/en/stable/running_mypy.html#reading-a-list-of-files-from-a-file). But not sure if mypy will be happy if this file does _not_ have correct or ignored typing hints.